### ISSUE TITLE
NSIS package fails to install system-wide in boring mode

### ISSUE THREAD

**Issue by lbssousa** [electron-userland/electron-builder#700]
* **Version**: 5.34.1

* **Target**: Windows (NSIS)

I've just built a NSIS package with the following options:
```
"oneClick": false,
"allowElevation": true
```

The installer asks me if I want to install per machine or per user. If I select **per machine**, it just skips the file copying step and goes through the final step, as if the installation was succeeded. If I select **per user**, it installs correctly.

If I include option `"perMachine": true` in my `package.json`, the build fails with the following message:
```
!insertmacro: macro named "setInstallModePerUser" not found!
Error in macro FUNCTION_INSTALL_MODE_PAGE_FUNCTION on macroline 18
Error in macro PAGE_INSTALL_MODE on macroline 17
!include: error in script: "boringInstaller.nsh" on line 21
Error in script "<stdin>" on line 9 -- aborting creation process
```

**Comment by develar** [comment]
How it is supposed to work — you set `perMachine: true` only for one-click installer. Not for boring installer because boring installer in any case "has" install mode page. 

Well, I see that such combination is requested and so, should be supported.

> it just skips the file copying step and goes through the final step

Works for me currently, app correctly installed.

**Comment by develar** [comment]
Fixed in 6.3.0
